### PR TITLE
Fix Composer configuration

### PR DESCRIPTION
**Pull request by szepeviktor**
Please see comments below.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		}
 	],
 	"require": {
-		"php": ">=7.4 || ^8.0",
```
Comment (szepeviktor) [inline composer.json:21]: `>=7.4` allows PHP 9.3 and 8.2 too.

File: composer.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
 		"optimize-autoloader": true,
 		"process-timeout": 0,
 		"platform": {
-			"php": "7.4"
+			"php": "7.4.30"
```
Comment (szepeviktor) [inline composer.json:50]: Let platform be an existing PHP version.

Comment (justlevine) [reply composer.json:50]: From the [docs](https://getcomposer.org/doc/06-config.md#platform), it seems `"7.4"` is valid.

If it isn't though, wouldn't we want at `"7.4.0"`? Any `7.4.x` version is supported and we want to make sure we grab deps that are compatible with all of them.

File: composer.json (hunk 1 above)
Comment (justlevine) [reply composer.json:21]: Great catch - Thanks!

File: composer.json (hunk 2 above)
Comment (szepeviktor) [reply composer.json:50]: This is not a requirement. It is a fake value that Composer uses.
It has nothing to do with your actual PHP version.
I leave it 100% up to you. I am not a user.

Comment (szepeviktor) [reply composer.json:50]: I set it to match the production environment.
https://www.php.net/ChangeLog-7.php
v7.4.30 was released a year ago.

Comment (szepeviktor) [reply composer.json:50]: I hope there is no Composer package that is not compatible with a higher path version of 7.4.x!

Comment (justlevine) [reply composer.json:50]: The issue is for a _user_ running 7.4.0, if a dep release requires 7.4.1 it shouldn't be installed.

Comment (szepeviktor) [reply composer.json]: All right.
Now it is 0.

**Pull request merged**